modules/gdscript/editor/script_templates/CharacterBody2D/godot_character_controller.gd

```diff
@@ -0,0 +1,44 @@
+# meta-description: Godot character controller for gravity games (platformer, ...)
+
+extends _BASE_
+
+
+@export_group("Kinematic")
+@export var walk_max_speed := 500.0
+@export var walk_acceleration := 1500.0
+@export var run_max_speed := walk_max_speed * 1.5
+@export var run_acceleration := walk_acceleration * 1.5
```

Comment:
I'm not sure that this group name is helpful. Walking and running are things that characters can do, not kinematic bodies in general. For example a moving platform does not have walking and running speeds. This is why KinematicBody was split into CharacterBody and AnimatableBody.

Reply:
@Calinou made a similar remark, what do you suggest? To delete it in 2D and rename it in 3D "character", or to delete it for both?

Reply:
I think it's just not necessary, we can delete it in both.